### ISSUE TITLE
fatal error: device_functions.hpp: No such file or directory

### ISSUE THREAD

**Issue by QuentinDela** [fireice-uk/xmr-stak#565]
# Basic information
GPU Nvidia M60

# Compile issues
Ubuntu Server 16.04 

  ```
$ sudo apt install libmicrohttpd-dev libssl-dev cmake build-essential libhwloc-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
build-essential is already the newest version (12.1ubuntu2).
libssl-dev is already the newest version (1.0.2g-1ubuntu4.10).
The following additional packages will be installed:
  cmake-data libgcrypt20-dev libgnutls-dev libgnutls28-dev libgnutlsxx28 libgpg-error-dev
  libhwloc-plugins libhwloc5 libidn11-dev libjsoncpp1 libltdl-dev libmicrohttpd10 libnuma-dev
  libp11-kit-dev libtasn1-6-dev libtasn1-doc libtool nettle-dev
Suggested packages:
  codeblocks eclipse ninja-build libgcrypt20-doc gnutls-doc gnutls-bin guile-gnutls
  libhwloc-contrib-plugins libtool-doc gfortran | fortran95-compiler gcj-jdk
The following NEW packages will be installed:
  cmake cmake-data libgcrypt20-dev libgnutls-dev libgnutls28-dev libgnutlsxx28 libgpg-error-dev
  libhwloc-dev libhwloc-plugins libhwloc5 libidn11-dev libjsoncpp1 libltdl-dev libmicrohttpd-dev
  libmicrohttpd10 libnuma-dev libp11-kit-dev libtasn1-6-dev libtasn1-doc libtool nettle-dev
0 upgraded, 21 newly installed, 0 to remove and 0 not upgraded.
Preparing to unpack .../libtool_2.4.6-0.1_all.deb ...
Unpacking libtool (2.4.6-0.1) ...
Selecting previously unselected package libhwloc5:amd64.
Preparing to unpack .../libhwloc5_1.11.2-3_amd64.deb ...
Unpacking libhwloc5:amd64 (1.11.2-3) ...
Selecting previously unselected package libnuma-dev:amd64.
Preparing to unpack .../libnuma-dev_2.0.11-1ubuntu1_amd64.deb ...
Unpacking libnuma-dev:amd64 (2.0.11-1ubuntu1) ...
Selecting previously unselected package libhwloc-dev:amd64.
Preparing to unpack .../libhwloc-dev_1.11.2-3_amd64.deb ...
Unpacking libhwloc-dev:amd64 (1.11.2-3) ...
Selecting previously unselected package libhwloc-plugins.
Preparing to unpack .../libhwloc-plugins_1.11.2-3_amd64.deb ...
Unpacking libhwloc-plugins (1.11.2-3) ...
Selecting previously unselected package libtasn1-doc.
Preparing to unpack .../libtasn1-doc_4.7-3ubuntu0.16.04.2_all.deb ...
Unpacking libtasn1-doc (4.7-3ubuntu0.16.04.2) ...
Processing triggers for man-db (2.7.5-1) ...
Processing triggers for libc-bin (2.23-0ubuntu9) ...
Processing triggers for install-info (6.1.0.dfsg.1-5) ...
Setting up cmake-data (3.5.1-1ubuntu3) ...
Setting up libjsoncpp1:amd64 (1.7.2-1) ...
Setting up cmake (3.5.1-1ubuntu3) ...
Setting up libgpg-error-dev (1.21-2ubuntu1) ...
Setting up libgcrypt20-dev (1.6.5-2ubuntu0.3) ...
Setting up libgnutlsxx28:amd64 (3.4.10-4ubuntu1.4) ...
Setting up nettle-dev (3.2-1ubuntu0.16.04.1) ...
Setting up libtasn1-6-dev:amd64 (4.7-3ubuntu0.16.04.2) ...
Setting up libp11-kit-dev:amd64 (0.23.2-5~ubuntu16.04.1) ...
Setting up libidn11-dev (1.32-3ubuntu1.2) ...
Setting up libgnutls-dev:amd64 (3.4.10-4ubuntu1.4) ...
Setting up libgnutls28-dev:amd64 (3.4.10-4ubuntu1.4) ...
Setting up libltdl-dev:amd64 (2.4.6-0.1) ...
Setting up libmicrohttpd10 (0.9.44+dfsg-1ubuntu2) ...
Setting up libmicrohttpd-dev (0.9.44+dfsg-1ubuntu2) ...
Setting up libtool (2.4.6-0.1) ...
Setting up libhwloc5:amd64 (1.11.2-3) ...
Setting up libnuma-dev:amd64 (2.0.11-1ubuntu1) ...
Setting up libhwloc-dev:amd64 (1.11.2-3) ...
Setting up libhwloc-plugins (1.11.2-3) ...
Setting up libtasn1-doc (4.7-3ubuntu0.16.04.2) ...
Processing triggers for libc-bin (2.23-0ubuntu9) ...

$ git clone https://github.com/fireice-uk/xmr-stak.git
Cloning into 'xmr-stak'...
remote: Counting objects: 3099, done.
remote: Total 3099 (delta 0), reused 0 (delta 0), pack-reused 3099
Receiving objects: 100% (3099/3099), 928.59 KiB | 0 bytes/s, done.
Resolving deltas: 100% (2164/2164), done.
Checking connectivity... done.

$ mkdir xmr-stak/build
$ cd xmr-stak/build
$ cmake ..
-- The C compiler identification is GNU 5.4.0
-- The CXX compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Set miner currency to 'monero' and 'aeon'
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Found CUDA: /usr/local/cuda (found suitable version "9.1", minimum required is "7.5")
-- Looking for CL_VERSION_2_0
-- Looking for CL_VERSION_2_0 - not found
-- Looking for CL_VERSION_1_2
-- Looking for CL_VERSION_1_2 - not found
-- Looking for CL_VERSION_1_1
-- Looking for CL_VERSION_1_1 - not found
-- Looking for CL_VERSION_1_0
-- Looking for CL_VERSION_1_0 - not found
-- Could NOT find OpenCL (missing:  OpenCL_LIBRARY OpenCL_INCLUDE_DIR)
CMake Error at CMakeLists.txt:259 (message):
  OpenCL NOT found: use `-DOpenCL_ENABLE=OFF` to build without OpenCL support
  for AMD gpu's
-- Configuring incomplete, errors occurred!
See also "/home/username/xmr-stak/build/CMakeFiles/CMakeOutput.log".
See also "/home/username/xmr-stak/build/CMakeFiles/CMakeError.log".

$ cmake -DOpenCL_INCLUDE_DIR=/usr/include/CL
CMake Error: The source directory "/home/username/xmr-stak/build" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.

$ sudo apt install ocl-icd-opencl-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following additional packages will be installed:
  opencl-headers
Recommended packages:
  libpoclu-dev
The following NEW packages will be installed:
  ocl-icd-opencl-dev opencl-headers
0 upgraded, 2 newly installed, 0 to remove and 0 not upgraded.
Need to get 58.5 kB of archives.
After this operation, 505 kB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://azure.archive.ubuntu.com/ubuntu xenial/main amd64 opencl-headers all 2.0~svn32091-2 [56.0 kB]
Get:2 http://azure.archive.ubuntu.com/ubuntu xenial/main amd64 ocl-icd-opencl-dev amd64 2.2.8-1 [2,524 B]
Fetched 58.5 kB in 0s (1,040 kB/s)
Selecting previously unselected package opencl-headers.
(Reading database ... 114734 files and directories currently installed.)
Preparing to unpack .../opencl-headers_2.0~svn32091-2_all.deb ...
Unpacking opencl-headers (2.0~svn32091-2) ...
Selecting previously unselected package ocl-icd-opencl-dev:amd64.
Preparing to unpack .../ocl-icd-opencl-dev_2.2.8-1_amd64.deb ...
Unpacking ocl-icd-opencl-dev:amd64 (2.2.8-1) ...
Setting up opencl-headers (2.0~svn32091-2) ...
Setting up ocl-icd-opencl-dev:amd64 (2.2.8-1) ...

$ cmake -DOpenCL_INCLUDE_DIR=/usr/lib/x86_64-linux-gnu/ .
-- Set miner currency to 'monero' and 'aeon'
-- Found OpenCL: /usr/lib/x86_64-linux-gnu/libOpenCL.so
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libssl.so;/usr/lib/x86_64-linux-gnu/libcrypto.so (found version "1.0.2g")
-- Configuring done
-- Generating done
-- Build files have been written to: /home/username/xmr-stak/build

$ make install
Scanning dependencies of target xmr-stak-c
[  2%] Building C object CMakeFiles/xmr-stak-c.dir/xmrstak/backend/cpu/crypto/c_jh.c.o
[  5%] Building C object CMakeFiles/xmr-stak-c.dir/xmrstak/backend/cpu/crypto/c_skein.c.o
[  8%] Building C object CMakeFiles/xmr-stak-c.dir/xmrstak/backend/cpu/crypto/c_blake256.c.o
[ 11%] Building C object CMakeFiles/xmr-stak-c.dir/xmrstak/backend/cpu/crypto/c_keccak.c.o
[ 14%] Building C object CMakeFiles/xmr-stak-c.dir/xmrstak/backend/cpu/crypto/c_groestl.c.o
[ 17%] Linking C static library bin/libxmr-stak-c.a
[ 17%] Built target xmr-stak-c
Scanning dependencies of target xmr-stak-backend
[ 20%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/jconf.cpp.o
[ 23%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/version.cpp.o
[ 26%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/backend/cpu/jconf.cpp.o
[ 29%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/backend/cpu/minethd.cpp.o
[ 32%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/backend/cpu/hwlocMemory.cpp.o
[ 35%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/backend/backendConnector.cpp.o
[ 38%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/backend/globalStates.cpp.o
[ 41%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/backend/cpu/crypto/cryptonight_common.cpp.o
[ 44%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/http/webdesign.cpp.o
[ 47%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/http/httpd.cpp.o
[ 50%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/misc/telemetry.cpp.o
[ 52%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/misc/executor.cpp.o
[ 55%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/misc/console.cpp.o
[ 58%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/misc/utility.cpp.o
[ 61%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/net/jpsock.cpp.o
[ 64%] Building CXX object CMakeFiles/xmr-stak-backend.dir/xmrstak/net/socket.cpp.o
[ 67%] Linking CXX static library bin/libxmr-stak-backend.a
[ 67%] Built target xmr-stak-backend
[ 70%] Building NVCC (Device) object CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/xmrstak_cuda_backend_generated_cuda_extra.cu.o
/home/username/xmr-stak/xmrstak/backend/nvidia/nvcc_code/cuda_extra.cu:9:32: fatal error: device_functions.hpp: No such file or directory
compilation terminated.
CMake Error at xmrstak_cuda_backend_generated_cuda_extra.cu.o.cmake:207 (message):
  Error generating
  /home/username/xmr-stak/build/CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/./xmrstak_cuda_backend_generated_cuda_extra.cu.o
CMakeFiles/xmrstak_cuda_backend.dir/build.make:70: recipe for target 'CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/xmrstak_cuda_backend_generated_cuda_extra.cu.o' failed
#include <stdio.h>
make[2]: *** [CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/xmrstak_cuda_backend_generated_cuda_extra.cu.o] Error 1
CMakeFiles/Makefile2:68: recipe for target 'CMakeFiles/xmrstak_cuda_backend.dir/all' failed
make[1]: *** [CMakeFiles/xmrstak_cuda_backend.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2

$ make install
[ 17%] Built target xmr-stak-c
[ 67%] Built target xmr-stak-backend
[ 70%] Building NVCC (Device) object CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/xmrstak_cuda_backend_generated_cuda_extra.cu.o
^CCMakeFiles/xmrstak_cuda_backend.dir/build.make:70: recipe for target 'CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/xmrstak_cuda_backend_generated_cuda_extra.cu.o' failed
make[2]: *** [CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/xmrstak_cuda_backend_generated_cuda_extra.cu.o] Interrupt
CMakeFiles/Makefile2:68: recipe for target 'CMakeFiles/xmrstak_cuda_backend.dir/all' failed
make[1]: *** [CMakeFiles/xmrstak_cuda_backend.dir/all] Interrupt
Makefile:127: recipe for target 'all' failed
make: *** [all] Interrupt


$ make install
[ 17%] Built target xmr-stak-c
[ 67%] Built target xmr-stak-backend
[ 70%] Building NVCC (Device) object CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/xmrstak_cuda_backend_generated_cuda_extra.cu.o
^CCMakeFiles/xmrstak_cuda_backend.dir/build.make:375: recipe for target 'CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/xmrstak_cuda_backend_generated_cuda_extra.cu.o' failed
make[2]: *** [CMakeFiles/xmrstak_cuda_backend.dir/xmrstak/backend/nvidia/nvcc_code/xmrstak_cuda_backend_generated_cuda_extra.cu.o] Interrupt
CMakeFiles/Makefile2:68: recipe for target 'CMakeFiles/xmrstak_cuda_backend.dir/all' failed
make[1]: *** [CMakeFiles/xmrstak_cuda_backend.dir/all] Interrupt
Makefile:127: recipe for target 'all' failed
make: *** [all] Interrupt



  ```

  ```
-- Set miner currency to 'monero' and 'aeon'
-- Configuring done
-- Generating done
-- Build files have been written to: /home/username/xmr-stak/build
-- Cache values
CMAKE_AR:FILEPATH=/usr/bin/ar
CMAKE_BUILD_TYPE:STRING=Release
CMAKE_COLOR_MAKEFILE:BOOL=ON
CMAKE_CXX_COMPILER:FILEPATH=/usr/bin/c++
CMAKE_CXX_FLAGS:STRING=
CMAKE_CXX_FLAGS_DEBUG:STRING=-g
CMAKE_CXX_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG
CMAKE_CXX_FLAGS_RELEASE:STRING=-O3 -DNDEBUG
CMAKE_CXX_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG
CMAKE_C_COMPILER:FILEPATH=/usr/bin/cc
CMAKE_C_FLAGS:STRING=
CMAKE_C_FLAGS_DEBUG:STRING=-g
CMAKE_C_FLAGS_MINSIZEREL:STRING=-Os -DNDEBUG
CMAKE_C_FLAGS_RELEASE:STRING=-O3 -DNDEBUG
CMAKE_C_FLAGS_RELWITHDEBINFO:STRING=-O2 -g -DNDEBUG
CMAKE_EXE_LINKER_FLAGS:STRING=
CMAKE_EXE_LINKER_FLAGS_DEBUG:STRING=
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_EXE_LINKER_FLAGS_RELEASE:STRING=
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_EXPORT_COMPILE_COMMANDS:BOOL=OFF
CMAKE_INSTALL_PREFIX:PATH=/home/username/xmr-stak/build
CMAKE_LINKER:FILEPATH=/usr/bin/ld
CMAKE_LINK_STATIC:BOOL=OFF
CMAKE_MAKE_PROGRAM:FILEPATH=/usr/bin/make
CMAKE_MODULE_LINKER_FLAGS:STRING=
CMAKE_MODULE_LINKER_FLAGS_DEBUG:STRING=
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_MODULE_LINKER_FLAGS_RELEASE:STRING=
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_NM:FILEPATH=/usr/bin/nm
CMAKE_OBJCOPY:FILEPATH=/usr/bin/objcopy
CMAKE_OBJDUMP:FILEPATH=/usr/bin/objdump
CMAKE_RANLIB:FILEPATH=/usr/bin/ranlib
CMAKE_SHARED_LINKER_FLAGS:STRING=
CMAKE_SHARED_LINKER_FLAGS_DEBUG:STRING=
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_SHARED_LINKER_FLAGS_RELEASE:STRING=
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_SKIP_INSTALL_RPATH:BOOL=NO
CMAKE_SKIP_RPATH:BOOL=NO
CMAKE_STATIC_LINKER_FLAGS:STRING=
CMAKE_STATIC_LINKER_FLAGS_DEBUG:STRING=
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL:STRING=
CMAKE_STATIC_LINKER_FLAGS_RELEASE:STRING=
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO:STRING=
CMAKE_STRIP:FILEPATH=/usr/bin/strip
CMAKE_VERBOSE_MAKEFILE:BOOL=FALSE
CPU_ENABLE:BOOL=ON
CUDA_64_BIT_DEVICE_CODE:BOOL=ON
CUDA_ARCH:STRING=30;35;37;50;52;60;61;62;70
CUDA_ATTACH_VS_BUILD_RULE_TO_CUDA_FILE:BOOL=ON
CUDA_BUILD_CUBIN:BOOL=OFF
CUDA_BUILD_EMULATION:BOOL=OFF
CUDA_COMPILER:STRING=nvcc
CUDA_CUDART_LIBRARY:FILEPATH=/usr/local/cuda/lib64/libcudart.so
CUDA_CUDA_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libcuda.so
CUDA_ENABLE:BOOL=ON
CUDA_GENERATED_OUTPUT_DIR:PATH=
CUDA_HOST_COMPILATION_CPP:BOOL=ON
CUDA_HOST_COMPILER:FILEPATH=/usr/bin/cc
CUDA_KEEP_FILES:BOOL=OFF
CUDA_NVCC_EXECUTABLE:FILEPATH=/usr/local/cuda/bin/nvcc
CUDA_NVCC_FLAGS:STRING=
CUDA_NVCC_FLAGS_DEBUG:STRING=
CUDA_NVCC_FLAGS_MINSIZEREL:STRING=
CUDA_NVCC_FLAGS_RELEASE:STRING=
CUDA_NVCC_FLAGS_RELWITHDEBINFO:STRING=
CUDA_PROPAGATE_HOST_FLAGS:BOOL=ON
CUDA_SDK_ROOT_DIR:PATH=CUDA_SDK_ROOT_DIR-NOTFOUND
CUDA_SEPARABLE_COMPILATION:BOOL=OFF
CUDA_SHOW_CODELINES:BOOL=OFF
CUDA_SHOW_REGISTER:BOOL=OFF
CUDA_TARGET_CPU_ARCH:STRING=
CUDA_TOOLKIT_INCLUDE:PATH=/usr/local/cuda/include
CUDA_TOOLKIT_ROOT_DIR:PATH=/usr/local/cuda
CUDA_TOOLKIT_TARGET_DIR:PATH=/usr/local/cuda
CUDA_USE_STATIC_CUDA_RUNTIME:BOOL=ON
CUDA_VERBOSE_BUILD:BOOL=OFF
CUDA_VERSION:STRING=9.1
CUDA_cublas_LIBRARY:FILEPATH=/usr/local/cuda/lib64/libcublas.so
CUDA_cudart_static_LIBRARY:FILEPATH=/usr/local/cuda/lib64/libcudart_static.a
CUDA_cufft_LIBRARY:FILEPATH=/usr/local/cuda/lib64/libcufft.so
CUDA_cupti_LIBRARY:FILEPATH=/usr/local/cuda/extras/CUPTI/lib64/libcupti.so
CUDA_curand_LIBRARY:FILEPATH=/usr/local/cuda/lib64/libcurand.so
CUDA_cusolver_LIBRARY:FILEPATH=/usr/local/cuda/lib64/libcusolver.so
CUDA_cusparse_LIBRARY:FILEPATH=/usr/local/cuda/lib64/libcusparse.so
CUDA_nppc_LIBRARY:FILEPATH=/usr/local/cuda/lib64/libnppc.so
CUDA_nppi_LIBRARY:FILEPATH=CUDA_nppi_LIBRARY-NOTFOUND
CUDA_npps_LIBRARY:FILEPATH=/usr/local/cuda/lib64/libnpps.so
CUDA_rt_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/librt.so
HWLOC:FILEPATH=/usr/lib/x86_64-linux-gnu/libhwloc.so
HWLOC_ENABLE:BOOL=ON
HWLOC_INCLUDE_DIR:PATH=/usr/include
MHTD:FILEPATH=/usr/lib/x86_64-linux-gnu/libmicrohttpd.so
MICROHTTPD_ENABLE:BOOL=ON
MTHD_INCLUDE_DIR:PATH=/usr/include
OPENSSL_CRYPTO_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libcrypto.so
OPENSSL_INCLUDE_DIR:PATH=/usr/include
OPENSSL_SSL_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libssl.so
OpenCL_ENABLE:BOOL=ON
OpenCL_INCLUDE_DIR:PATH=/usr/lib/x86_64-linux-gnu
OpenCL_LIBRARY:FILEPATH=/usr/lib/x86_64-linux-gnu/libOpenCL.so
OpenSSL_ENABLE:BOOL=ON
PKG_CONFIG_EXECUTABLE:FILEPATH=/usr/bin/pkg-config
XMR-STAK_COMPILE:STRING=native
XMR-STAK_CURRENCY:STRING=all
XMR-STAK_LARGEGRID:BOOL=ON
XMR-STAK_THREADS:STRING=0
  ```

**Comment by QuentinDela** [comment]
I tired the solution from [there](https://github.com/xmrig/xmrig-nvidia/issues/64) but it didnt work.

**Comment by QuentinDela** [comment]
[This](https://github.com/xmrig/xmrig-nvidia/issues/64) solved my issue.

**Comment by psychocrypt** [comment]
solved in #532
